dGame/dGameMessages/GameMessages.cpp

```diff
@@ -5086,7 +5088,8 @@ void GameMessages::HandleNotifyServerLevelProcessingComplete(RakNet::BitStream*
 	wss << "name=0:";
 	wss << character->GetName();
 
-	std::u16string attrs = GeneralUtils::ASCIIToUTF16(wss.str());
+	// FIXME: only really need utf8 conversion for the name, so move that up?
```

Comment:
Technically speaking this whole stringstream should be re-written to LDF instead of a hard coded stream like this. Would be more descriptive of the actual fix.

Reply:
Good point!

Reply:
Just to note, there are several places that need to be updated to use proper LDF instead of hardcoded streams